db/migrate/20220310194756_remove_court_date_from_casa_cases.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveCourtDateFromCasaCases < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured { remove_column :casa_cases, :court_date }
```

Comment:
This is dangerous... let me think about it for a minute :)

Reply:
Ya.. I suppose we can leave it in in case it's used anywhere downstream and just eliminate calls to the attribute. We can also raise if a dev tries to call casa_case.court_date moving forward

Reply:
Let's use https://www.bigbinary.com/blog/rails-5-adds-active-record-ignored-columns first and then do the migration deleting in a separate later PR